code/game/turfs/flooring/_flooring.dm

```diff
@@ -391,3 +391,6 @@ var/global/list/flooring_cache = list()
 /// contaminant is, optionally, the material of the coating that wants to be added.
 /decl/flooring/proc/can_show_coating_footprints(turf/target, decl/material/contaminant)
 	return TRUE
+
+/decl/flooring/proc/get_vehicle_transit_delay(obj/vehicle/vehicle)
+	return initial(vehicle.speed_mod)
```

Comment:
undecided on if this should be `vehicle::speed_mod` or not. i also kind of would prefer a separate `base_speed` variable instead of using initial, because relying on initial value tends to make future-proofing a little painful. not a required change though

Reply:
Ended up on redoing how base speed was calculated to clean it up a bit. Included a `base_speed` value now.